src/main/java/com/exasol/adapter/AdapterRegistry.java

```diff
@@ -65,7 +65,10 @@ public void registerAdapterFactory(final String adapterName, final AdapterFactor
      */
     public VirtualSchemaAdapter getAdapterForName(final String name) {
         if (hasAdapterWithName(name)) {
-            return this.registeredFactories.get(name).createAdapter();
+            AdapterFactory factory = this.registeredFactories.get(name);
+            LOGGER.config("Virtual Schema Adapter name: " + factory.getAdapterName());
```

Comment:
This is not the right place and things like this should also always be in a single message.
Add it to
`com.exasol.adapter.RequestDispatcher.logVersionInformation()` please.
You have to move the call a few lines down to the point where the adapter is loaded then.